content: add november meeting (#8)
ockam-network/meetings: 2021/11-17.md

- **Where**: [Zoom Video Call](https://ockam.zoom.us/j/93012459605?pwd=VG1YU0NUWExtd25OL0dma283cUZ0Zz09)
- **When**: Nov 17th, 1pm-2pm Pacific Local Time

### Registration

None required. The meeting is open to all members in our community.

## Logistics

* The meeting will be on a Zoom video conference.
* Etiquette dictates that you join with video enabled - although it is not explicitly required.
* Use **Mute** as a default. Press `space bar` to talk/unmute.
* Use the chat window for dialog as much as possible.
* [Ockam Community on GitHub Discussions](https://github.com/ockam-network/ockam/discussions)
* [Ockam Community - Code Of Conduct Covenant](https://www.ockam.io/learn/guides/team/conduct/)
* Community Calls will be *recorded* and will be made available as soon as possible in this repo.
* This meeting will have a hard stop at 2pm PDT.

## Agenda items

1. Opening, welcome and roll call
    1. Opening of the meeting by Matthew Gregory
        1. [How to join The Team](https://www.ockam.io/learn/guides/team/join_us/)
        1. [Open Roles / Active Hiring](https://www.ockam.io/team#open-roles)
    1. Introduction of attendees
    1. Find volunteers for note taking

> 1. A deep dive into how Ockam Inlets and Outlets work and how they help provide trust to any application layet data transmission

Reviewer: ```suggestion
1. How to end-to-end encrypt all application layer communication - a dive into Ockam Inlets and Outlets
```
